fhir-install/docker/Dockerfile-ol

```diff
@@ -0,0 +1,52 @@
+# ----------------------------------------------------------------------------
+# (C) Copyright IBM Corp. 2016, 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
+
+FROM openliberty/open-liberty as base
+
+ENV LICENSE accept
+
+USER root
+RUN \
+  yum update -y && \
+  yum install -y unzip nmap-ncat && \
+  yum -y clean all
+
+USER 1001
+COPY volumes/dist /dist
+COPY volumes/scripts  /scripts
+RUN unzip -qq /dist/fhir-server-distribution.zip -d /tmp && \
+    /tmp/fhir-server-dist/install-fhir.sh /opt/ol/wlp && \
+    rm -fr /opt/ol/wlp/usr/servers/fhir-server/config/* && \
+    cp -pr /dist/config/* /opt/ol/wlp/usr/servers/fhir-server/config/ && \
+    cp -pr /dist/userlib/* /opt/ol/wlp/usr/servers/fhir-server/userlib/
+
+
```

Comment:
extra line

Reply:
purposeful to separate the second FROM from the top half...something we do to reduce image size